FirebaseDatabase/Tests/Unit/Swift/DatabaseAPITests.swift

```diff
@@ -0,0 +1,683 @@
+//
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// MARK: This file is used to evaluate the experience of using Firebase APIs in Swift.
+
+import Foundation
+
+import FirebaseCore
+import FirebaseDatabase
+
+final class DatabaseAPITests {
+  func usage() {
+    // MARK: - Database
+
+    var url = "url"
+    let path = "path"
+    let host = "host"
+    let port = 0
+    let yes = true
+
+    // Retrieve Database Instance
+    var database = Database.database()
+    _ = database as Database
```

Comment:
Looking at this again, since we use `database` more than once, I think we should should add line 36's type info to line 35. So:
```swift
var database: Database = Database.database()
```